CenterlineMetrics/CenterlineMetrics.py

```diff
@@ -239,20 +411,62 @@ def updateUIWithMetrics(self, value):
       self.ui.derivedDiameterValueLabel.setText(derivedDiameterStr)
     else:
       self.ui.derivedDiameterValueLabel.setText("")
-
+    # Orientation
+    self.updateSliceViewOrientationMetrics()
+    
+  def updateSliceViewOrientationMetrics(self):
+    """
```

Comment:
Please remove this comment and instead submit an issue to discuss/resolve this.

Reply:
Done. I'll open a new issue ASAP.